mobile/lib/src/services/device_locator_service.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:async';
+
+import 'package:polydodo/src/domain/acquisition_device/acquisition_device.dart';
+import 'package:polydodo/src/domain/acquisition_device/device_type.dart';
+import 'package:polydodo/src/domain/acquisition_device/i_acquisition_device_repository.dart';
+import 'package:polydodo/src/domain/acquisition_device/i_device_locator_service.dart';
+import 'package:polydodo/src/infrastructure/connection_repositories/bluetooth_repository.dart';
+import 'package:polydodo/src/infrastructure/connection_repositories/serial_repository.dart';
+
+class DeviceLocatorService implements IDeviceLocatorService {
```

Comment:
The service does not need to be implemented. A domain service can use domain objects only by using the repository interfaces.